src/js/view/controls/controlbar.js

```diff
@@ -139,11 +141,18 @@ export default class Controlbar {
             volumeTooltip = new VolumeTooltip(_model, 'jw-icon-volume', vol,
                 cloneIcons('volume-0,volume-50,volume-100'));
 
-            const volumeTooltipEl = volumeTooltip.element();
+            volumeTooltipEl = volumeTooltip.element();
             setAttribute(volumeTooltipEl, 'aria-valuemin', 0);
             setAttribute(volumeTooltipEl, 'aria-valuemax', 100);
             setAttribute(volumeTooltipEl, 'aria-orientation', 'vertical');
             setAttribute(volumeTooltipEl, 'role', 'slider');
+            muteTip = SimpleTooltip(volumeTooltipEl, 'mutetooltip', localization.mute);
+            volumeTooltipEl.removeEventListener('mouseover', muteTip.open);
+            _model.change('mute', (model, muted) => {
+                const muteText = muted ? localization.unmute : localization.mute;
+                muteTip.setText(muteText);
+                setAttribute(volumeTooltipEl, 'aria-label', muteText);
+            }, this);
```

Comment:
Could we just push `volumeTooltip` into `menus` here instead of that silly filter method below?